docs/blog/posts/2024-04-25-yen_gpu.md

```diff
@@ -261,69 +249,56 @@ Accuracy on the test set: 97 %
 ```
 
 
-### Make PyTorch into a Jupyter Kernel
-We can also add this environment to the interactive Yen's JupyterHub. Note that even though PyTorch will fall back to the CPU if GPU is not available,
-deep learning and machine learning is much more efficient on GPU than CPU so you should not use the interactive yens for model training but
-use the notebooks for visualization or other pre- or post-training tasks.
+#### Make PyTorch into a Jupyter Kernel
 
-```bash
-$ module purge
-$ ml anaconda3
-```
+We can also add this virtual environment to the [interactive Yen's JupyterHub](/_getting_started/jupyter/){:target="_blank"}. Although PyTorch will fall back to use CPU on JupyterHub since GPU is not available, you can still utilize JupyterHub notebooks for visualization and other pre-/post-training tasks. For model training and inference, it is better to utilize the GPU nodes on `yen-slurm`.
 
-List all available kernels:
-```bash
-$ jupyter kernelspec list
+You can start by listing all of your available JupyterHub kernels:
+```bash title="Terminal Command"
+jupyter kernelspec list
 ```
 
-Load `pytorch` module and make `venv` into a JupyterHub kernel:
-```bash
-$ ml pytorch
-$ python -m ipykernel install --user --name pytorch201 --display-name 'PyTorch 2.0.1'
+To add the PyTorch environment, load the `pytorch` module and then create a JupyterHub kernel from the `venv` using the following commands:
+```bash title="Terminal Command"
+ml pytorch
+python -m ipykernel install --user --name pytorch201 --display-name 'PyTorch 2.0.1'
 ```
 
-Launch <a href="/yen/webBasedCompute.html" target="_blank">JupyterHub</a> on the *same* interactive yen that you have
-loaded the `pytorch` module. You should see a new `PyTorch 2.0.1` Notebook kernel you can start up.
+Now launch JupyterHub on the *same* interactive yen that you have loaded the `pytorch` module on. When you launch a new Python notebook, you should see the just created `PyTorch 2.0.1` notebook kernel that you can start up.
```

Comment:
i think we can use the kernel on any node now. Don't need to start it on the same yen